W-19708281-govcloud-salesforce-kt (#61)
mulesoft/docs-govcloud: modules/ROOT/pages/index.adoc

[NOTE]
include::access-management::partial$include-gov-cloud-note.adoc[tag=govcloud]
[NOTE]

> The Salesforce menu option in Access Management is not available in Government Cloud.

Reviewer: ```suggestion
The Salesforce menu option in Access Management isn't available in Government Cloud.
```
